fv3core/pace/fv3core/stencils/divergence_damping.py

```diff
@@ -236,11 +236,7 @@ def redo_divg_d(
             divg_d = divg_d + uc
 
     with computation(PARALLEL), interval(...):
-        # TODO: this does the wrong thing when stretched_grid is True,
```

Comment:
Did this actually get fixed for stretched grids?

Reply:
I double checked it, and as far as I could tell there's no bug here. It was already added in the `dd8` calculation.